lightning/restore/restore_test.go

```diff
@@ -232,17 +236,16 @@ func (s *restoreSuite) TestIncreaseGCLifeTimeFail(c *C) {
 	}
 	wg.Wait()
 
-	// validate no more redundant GC recover
-	err = mock.ExpectationsWereMet()
-	c.Assert(err, ErrorMatches, "there is a remaining expectation.*\n.*\n.*\n.*\n.*\n.*\n.*")
+	_, err = db.Exec("\\QUPDATE mysql.tidb SET VARIABLE_VALUE = ? WHERE VARIABLE_NAME = 'tikv_gc_life_time'\\E", "10m")
+	c.Assert(err, IsNil)
 
-	mock.ExpectClose()
+	c.Assert(db.Close(), IsNil)
+	err = mock.ExpectationsWereMet()
```

Comment:
Assert it is not nil ?

Reply:
`db.Close()` returns `err`, so it's short for error check.

BTW I noticed L243 didn't have a check...

Reply:
@lance6716 sorry typo 😅, I do mean line 243 should be asserted to be nil.